src/OpenSage.Game/Audio/AudioSystem.cs

```diff
@@ -185,9 +185,13 @@ private void UpdateListener(Camera camera)
 
         public void PlayAudioEvent(GameObject emitter, BaseAudioEventInfo baseAudioEvent)
         {
+            return;
```

Comment:
Bfme crashes at the moment when some specific units are selected, @feliwir will invesitgate this further.

Reply:
> Bfme crashes at the moment when some specific units are selected, @feliwir will invesitgate this further.

Crashes are likely to be related to their commandsets and specifically to certain commandbuttons that those specific units have.

Reply:
No, it’s due to the 3d soundsystem, so @Tarcontar is correct

Reply:
@Tarcontar Could you make this call the regular **PlayAudioEvent** for now and add a TODO please? I'll investigate once i have time